package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-babel-plugin",
-  "version": "2.0.1",
```

Comment:
Please leave `version` as-is, we will bump this version when published.

Reply:
Right, updated!